markup/pandoc/convert_test.go

```diff
@@ -25,18 +25,114 @@ import (
 	qt "github.com/frankban/quicktest"
 )
 
-func TestConvert(t *testing.T) {
+func SetupTestConverter(t *testing.T) (*qt.C, converter.Converter, converter.ProviderConfig) {
```

Comment:
This can be unexported?

Reply:
I didn't know that concept, as I haven't done much programming in Go, thank you very much!